src/main/java/com/mraof/minestuck/data/BoondollarPricingProvider.java

```diff
@@ -44,112 +45,112 @@ public BoondollarPricingProvider(DataGenerator dataGenerator, String modid)
 	
 	protected void registerPricings()
 	{
-		add(ONION, 9, 15);
-		add(JAR_OF_BUGS, 12, 18);
-		add(BUG_ON_A_STICK, 4, 6);
-		add(CONE_OF_FLIES, 4, 6);
-		add(GRASSHOPPER, 90, 110);
-		add(SALAD, 10, 14);
-		add(CHOCOLATE_BEETLE, 30, 35);
-		add(DESERT_FRUIT, 2, 6);
-		add(GLOWING_MUSHROOM.get(), 10, 15);
-		add(COLD_CAKE.get(), 400);
-		add(BLUE_CAKE.get(), 400);
-		add(HOT_CAKE.get(), 400);
-		add(RED_CAKE.get(), 400);
-		add(FUCHSIA_CAKE.get(), 1500);
-		add(ROCK_COOKIE, 15, 20);
-		add(STRAWBERRY_CHUNK, 100, 150);
-		add(TAB, 200);
-		add(ORANGE_FAYGO, 100);
-		add(CANDY_APPLE_FAYGO, 100);
-		add(FAYGO_COLA, 100);
-		add(COTTON_CANDY_FAYGO, 100);
-		add(CREME_SODA_FAYGO, 100);
-		add(GRAPE_FAYGO, 100);
-		add(MOON_MIST_FAYGO, 100);
-		add(PEACH_FAYGO, 100);
-		add(REDPOP_FAYGO, 100);
-		add(GOLD_SEEDS.get(), 300, 400);
-		add(APPLE_CAKE.get(), 100, 140);
-		add(IRRADIATED_STEAK, 70, 80);
-		add(CANDY_CORN, 100, 150);
-		add(BUILD_GUSHERS, 90, 120);
-		add(AMBER_GUMMY_WORM, 125, 150);
-		add(CAULK_PRETZEL, 125, 150);
-		add(CHALK_CANDY_CIGARETTE, 125, 150);
-		add(IODINE_LICORICE, 125, 150);
-		add(SHALE_PEEP, 125, 150);
-		add(TAR_LICORICE, 125, 150);
-		add(COBALT_GUM, 150, 180);
-		add(MARBLE_JAWBREAKER, 150, 180);
-		add(MERCURY_SIXLETS, 150, 180);
-		add(QUARTZ_JELLY_BEAN, 150, 180);
-		add(SULFUR_CANDY_APPLE, 150, 180);
-		add(AMETHYST_HARD_CANDY, 175, 210);
-		add(GARNET_TWIX, 175, 210);
-		add(RUBY_CROAK, 175, 210);
-		add(RUST_GUMMY_EYE, 175, 210);
-		add(DIAMOND_MINT, 200, 240);
-		add(GOLD_CANDY_RIBBON, 200, 240);
-		add(URANIUM_GUMMY_BEAR, 200, 240);
-		add(ARTIFACT_WARHEAD, 225, 270);
-		add(ZILLIUM_SKITTLES, 250, 300);
+		add(ONION.get(), 9, 15);
+		add(JAR_OF_BUGS.get(), 12, 18);
+		add(BUG_ON_A_STICK.get(), 4, 6);
+		add(CONE_OF_FLIES.get(), 4, 6);
+		add(GRASSHOPPER.get(), 90, 110);
+		add(SALAD.get(), 10, 14);
+		add(CHOCOLATE_BEETLE.get(), 30, 35);
+		add(DESERT_FRUIT.get(), 2, 6);
+		add(MSBlocks.GLOWING_MUSHROOM.get(), 10, 15);
```

Comment:
Since MSItems now also hold references to block items, I'd suggest removing the static import of `MSBlocks`, and instead usie the already imported references from `MSItems` instead